brave/browser-laptop
# Disable websql support

Fix #15337 

Test plan:
1. open devtools
2. confirm that `window.openDatabase` is undefined 
3. other client-side storage mechanisms mentioned in https://www.html5rocks.com/en/tutorials/offline/storage/, such as window.localStorage, should be defined


## Submitter Checklist:


## Test Plan:


## Reviewer Checklist:


Tests
